README.md

```diff
@@ -194,6 +194,37 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
 
 You can find documentation regarding our HTTP interface [here](http_iface_docs.md).
 
+# Available Trusted Peers
+
+| type |                               domain name | notes |
+| ---- | ----------------------------------------- | ----- |
+| pool |                        `pool.arweave.xyz` | global pool containing all trusted peers |
```

Comment:
We should probably find a different name for this since "pool" is already associated with "mining pool".

Maybe cluster? Group? ip_set?

Reply:
Ah and I'd move this documentation into the main docs so that we don't have to worry about keeping it up to date in two locations: https://github.com/ArweaveTeam/docs.arweave.org-info/blob/master/mining/mining-guide.md#trusted-peers

But feel free to create a new section in the docs too - it doesn't have to be buried inside the mining guide, we can link out a separate chapter if needed

Reply:
I'd love to not use "pool" - when miners set up their configuratin they may specify a pool address (referring to a mining pool) and set of trusted peers. If those trusted peers also use the word "pool" it's just going to make everything very confusing. I can already see the endless Discord questions "Can I use the pool.xxx.xyz peers even if I'm not in a mining pool?" - "I don't want to join a mining pool, does anyone have a set of trusted peers that aren't attached to a pool?" etc etc

basically any word other than pool. Please please please.